eng/pipelines/test-unix-job.yml

```
@@ -36,20 +36,29 @@ jobs:
       vmImage: ${{ parameters.vmImageName }}
   timeoutInMinutes: 40
   steps:
-    - script: ./eng/build.sh --ci --restore --prepareMachine --binaryLog --configuration ${{ parameters.configuration }}
-      displayName: Restore
+    - checkout: none
 
     - task: DownloadPipelineArtifact@2
+      displayName: Download test payload
```

```suggestion
      displayName: Download Test Payload
```

Don't really care which capitilazation style we choose just bugging me that it wasn't consistent.